src/runtime/types.ts

```diff
@@ -548,6 +548,12 @@ export interface SecondarySignInOptions extends Record<string, unknown> {
    * @default false
    */
   external?: boolean
+  /**
+   * Whether `getSession` is called. If set to false, you have to manually call `getSession` to get the session data.
+   *
+   * @default true
+   */
+  withSession?: boolean
```

Comment:
I am actually not in favor of this name, as I used it when testing your PR and it doesn't "click". Existing options use the imperative forms and I think we should stick to it for the sake of consistency (e.g. `redirect`, `preventLoginFlow`).

Another problem I see with `withSession` is that it may confuse users as to what it means. I propose a more explicit

```suggestion
  callGetSession?: boolean
```

Reply:
There was an earlier thread with Zoey about this. We ended up with `withSession`. I think it's fine to change, I will do it. See https://github.com/sidebase/nuxt-auth/pull/702/files/e59026bf2b8492e382b04ba8f1f7e12bf072f601#diff-dd4f118f099f41a809eb53d231259e96a9818b3d7f734b4b671de323ac718685